WooCommerce/UITestsFoundation/Screens/Orders/SingleOrderScreen.swift

```diff
@@ -8,11 +8,24 @@ public final class SingleOrderScreen: ScreenObject {
 
     init(app: XCUIApplication = XCUIApplication()) throws {
         try super.init(
-            expectedElementGetters: [ { $0.staticTexts["summary-table-view-cell-title-label"] } ],
+            expectedElementGetters: [{ $0.staticTexts["summary-table-view-cell-title-label"]} ],
             app: app
         )
     }
 
+    @discardableResult
+    public func verifySingleOrder(order: OrderData) throws -> Self {
+        app.assertTextVisibilityCount(textToFind: order.status, expectedCount: 1) //checks that the Mock order status appears on the screen
+        app.assertTextVisibilityCount(textToFind: order.billing.first_name, expectedCount: 2) // Check that the Mock first name ("Mira") appears twice - because this order has both billing and shipping details
```

Comment:
In this case the explanation why the first name is expected to be shown twice might be helpful, so I'd keep it, but I'd make the comment shorter (we don't really need to mention the actual mock values to have the comment idea expressed):

```suggestion
        // Check that the first name appears twice - because the order screen has both billing and shipping details
        app.assertTextVisibilityCount(textToFind: order.billing.first_name, expectedCount: 2)
```